ibm/mas_devops/roles/suite_install/defaults/main.yml

```diff
@@ -116,3 +116,6 @@ mas_img_pull_policy: "{{ lookup('env', 'MAS_IMG_PULL_POLICY') }}"
 # Custom Labels
 # -----------------------------------------------------------------------------
 custom_labels: "{{ lookup('env', 'CUSTOM_LABELS') | default(None, true) | string | ibm.mas_devops.string2dict() }}"
+
+# Dualstack required?
+ipv6_test: "{{ lookup('env', 'IPV6_TEST') | default('false', true) }}"
```

Comment:
Similar to the same var name in ocp_provision, we should call this enable_dualstack or something similar. It should also be added to the docs but with a statement that this is not supported yet.